std/getopt.d

```diff
@@ -611,6 +612,13 @@ private template optionValidator(A...)
     static assert(optionValidator!(S,S,S,P) != "");
 }
 
+unittest
+{
+    bool opt;
+    string[] args = ["program", "-a"];
+    static assert( is(typeof(getopt(args, config.passThrough, 'a', &opt))));
```

Comment:
`static assert( is(typeof(...)))` is redundant and only serves to obscure why the check fails.